training/src/app/chuck-norris-main/chuck-norris-main.component.css

```diff
@@ -0,0 +1,117 @@
+
+.top-bar > #top-bar-img {
```

Comment:
the > sign is useful when u need to find for example all span elements inside,
in your case there is not much of a value, since u are calling an id class.
so simply do :
.top-bar {
  #top-bar-img {...}
}

also for other places in this file